test/export/test_db.py

```diff
@@ -66,29 +66,6 @@ def test_exportdb_not_supported(self, name: str, case: ExportCase) -> None:
                 dynamic_shapes=case.dynamic_shapes,
             )
 
-    @parametrize(
```

Comment:
Not sure what to do here. The type reflection is currently the only rewrite case. With that fixed, the parametrization fails, because we don't have anything to parametrize over. However, we probably want to retain the test for the next time someone registers a rewrite case.

One option could be to provide a sentinel value in case the parametrization is empty and fail the test in case we get anything but the sentinel. SOmething like

```python
@parametrize("name,rewrite_case", [...] or [(None, None)])
def test_exportdb_not_supported_rewrite(self, name, rewrite_case):
    assert name is None and rewrite_case is None, "Please remove the default above"
    return
    ...
```